packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -96,7 +96,7 @@ export default decorate([
       async create() {
         const { isPoolCreated, installedTemplates } = this.state
         const { name } = this.props
-        if (getXoaPlan(process.env.XOA_PLAN) === 'Community') {
+        if (getXoaPlan(+process.env.XOA_PLAN) === 'Community') {
```

Comment:
This is the default value.

```suggestion
        if (getXoaPlan(+process.env.XOA_PLAN) === 'Community') {
```